### DISCUSSION TITLE
How to get DateTimeOffset.Now when patching with RQL

### DISCUSSION THREAD

**Discussion by JoshClose** [ravendb/ravendb#13907] [Q&A] [answered]
Is there a way to get the `DateTimeOffset.Now.ToString("o")` format in an RQL patch?

**Comment by JoshClose** [comment, marked as answer]
Looks like I can use `new Date().toISOString()`. Would there be an issue with that?

**Reply by ayende** [reply]
Yes, that is the proper way to do that.
